packages/projects-docs/pages/learn/_meta.json

```diff
@@ -1,11 +1,27 @@
 {
-  "introduction": "Introduction",
-  "getting-started": "Getting Started",
-  "sandboxes": "Sandboxes",
+  "index": "Introduction",
+  "getting-started": "Guides",
+  "-- cloud-development": {
+    "type": "separator",
+    "title": "Cloud Development"
+  },
   "repositories": "Repositories",
+  "sandboxes": "Sandboxes",
```

Comment:
I would rename this to "Cloud Sandboxes" for the time being, as it might confuse existing users. Later on, we can probably safely rename it back to "Sandboxes".

Reply:
sounds good